lib/countdown.js

```diff
@@ -274,6 +274,7 @@ class CountdownTimer {
     }
   }
 
+
```

Comment:
Was this intentional?

Reply:
Addressed in 

https://github.com/gadenbuie/countdown/pull/35/commits/e0ac982d57a17fb3f3229d50e4ba24b838751259